bin/start_backend.sh

```diff
@@ -80,18 +83,26 @@ if [ $? -ne 0 ]; then
     exit 1
 fi
 
-# JEMALLOC enable DEBUG 
-# export JEMALLOC_CONF="junk:true,tcache:false,prof:true"
+# enable jemalloc
+JEMALLOC_LIB=$STARROCKS_HOME/lib/libjemalloc.so
+ln -s -f $STARROCKS_HOME/lib/libjemalloc.so.2 $JEMALLOC_LIB
+export LD_LIBRARY_PATH=$STARROCKS_HOME/lib:$LD_LIBRARY_PATH
+
 # Set JEMALLOC_CONF environment variable if not already set
 if [[ -z "$JEMALLOC_CONF" ]]; then
-    if [ ${RUN_CHECK_MEM_LEAK} -eq 1 ] ; then
-      export JEMALLOC_CONF="percpu_arena:percpu,oversize_threshold:0,muzzy_decay_ms:5000,dirty_decay_ms:5000,metadata_thp:auto,background_thread:true,prof:true,prof_active:true,prof_leak:true,lg_prof_sample:0,prof_final:true"
+    # JEMALLOC enable DEBUG 
+    if [ ${RUN_JEMALLOC_DEBUG} -eq 1 ] ; then
+        ln -s -f $STARROCKS_HOME/lib/libjemalloc-dbg.so.2 $JEMALLOC_LIB
+        export JEMALLOC_CONF="junk:true,tcache:false,prof:true"
```

Comment:
can this be running the same conf as line 98 with additional junk:true settings?  It is so to mimic as much as possible in an online env (possibly a prod env), avoiding unexpected difference when reproducing issues.

Reply:
If cpu arena cache is turned on, the probability of detecting the problem is reduced.

Reply:
that is not be necessary true.

Reply:
https://github.com/jemalloc/jemalloc/wiki/Use-Case%3A-Find-a-memory-corruption-bug

Disabling thread caching (MALLOC_CONF=tcache:false) tends to make jemalloc's internal assertions much more effective at catching application errors, especially double frees.